src/Affects/Mails/Events/ConfigureMails.php

```diff
@@ -46,9 +45,10 @@ public function __construct(Switched $event, Mailer $mailer)
      *
      * @return self
      */
-    public function replaceSwiftMailer(Swift_Transport $transport)
+    public function replaceSymfonyTransport(TransportInterface $transport)
```

Comment:
just replaceTransport would be enough for me

Reply:
The Laravel 9 Methods all seem to say Symfony in them so perhaps we should leave it as is?

`$this->mailer->setSymfonyTransport($transport);`

```
$this->withSymfonyMessage(function (\Symfony\Component\Mime\Email $message) {
$message->getHeaders()->addTextHeader('TEST', 'TEST');
}
```